packages/app-store/stripepayment/lib/PaymentService.ts

```diff
@@ -92,11 +159,72 @@ export class PaymentService implements IAbstractPaymentService {
           fee: paymentFee,
           refunded: false,
           success: false,
+          paymentOption: paymentOption || "ON_BOOKING",
+        },
+      });
+      if (!paymentData) {
+        throw new Error();
+      }
+      return paymentData;
+    } catch (error) {
+      console.error(error);
+      throw new Error("Payment could not be created");
+    }
+  }
+
+  async chargeCard(payment: Payment, bookingId: Booking["id"]): Promise<Payment> {
```

Comment:
This method should charge a stored payment method, so the user does not need to enter their payment method again